src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+                }
+              this.setTransitioning(!0);
+              this._element.style[i] = "";
+              var c = l.getTransitionDurationFromElement(this._element);
+              e(this._element)
+                .one(l.TRANSITION_END, function () {
+                  t.setTransitioning(!1),
+                    e(t._element)
+                      .removeClass("collapsing")
+                      .addClass("collapse")
+                      .trigger("hidden.bs.collapse");
+                })
+                .emulateTransitionEnd(c);
+            }
+          }
+        }),
+        (n.setTransitioning = function (t) {
+          this._isTransitioning = t;
+        }),
+        (n.dispose = function () {
+          e.removeData(this._element, "bs.collapse"),
+            (this._config = null),
+            (this._parent = null),
+            (this._element = null),
+            (this._triggerArray = null),
+            (this._isTransitioning = null);
+        }),
+        (n._getConfig = function (t) {
+          return (
+            ((t = s(s({}, T), t)).toggle = Boolean(t.toggle)),
+            l.typeCheckConfig(w, t, C),
+            t
+          );
+        }),
+        (n._getDimension = function () {
+          return e(this._element).hasClass("width") ? "width" : "height";
+        }),
+        (n._getParent = function () {
+          var n,
+            i = this;
+          l.isElement(this._config.parent)
+            ? ((n = this._config.parent),
+              "undefined" != typeof this._config.parent.jquery &&
+                (n = this._config.parent[0]))
+            : (n = document.querySelector(this._config.parent));
+          var o =
+              '[data-toggle="collapse"][data-parent="' +
+              this._config.parent +
+              '"]',
+            r = [].slice.call(n.querySelectorAll(o));
+          return (
+            e(r).each(function (e, n) {
+              i._addAriaAndCollapsedClass(t._getTargetFromElement(n), [n]);
+            }),
+            n
+          );
+        }),
+        (n._addAriaAndCollapsedClass = function (t, n) {
+          var i = e(t).hasClass("show");
+          n.length &&
+            e(n).toggleClass("collapsed", !i).attr("aria-expanded", i);
+        }),
+        (t._getTargetFromElement = function (t) {
+          var e = l.getSelectorFromElement(t);
+          return e ? document.querySelector(e) : null;
+        }),
+        (t._jQueryInterface = function (n) {
+          return this.each(function () {
+            var i = e(this),
+              o = i.data("bs.collapse"),
+              r = s(s(s({}, T), i.data()), "object" == typeof n && n ? n : {});
+            if (
+              (!o &&
+                r.toggle &&
+                "string" == typeof n &&
+                /show|hide/.test(n) &&
+                (r.toggle = !1),
+              o || ((o = new t(this, r)), i.data("bs.collapse", o)),
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)